bot/src/bot.js

```diff
@@ -28,6 +28,25 @@ function saveUpdate(db, author, channel, text) {
   });
 }
 
+const createDateRe = /create a report for ([0-9]{4}-[0-9]{2}-[0-9]{2})/;
```

Comment:
I've been thinking about this and came to a conclusion to disagree with myself ;)  how about we actually use a forward slash to denote special commands for now?